src/main/java/duke/task/Task.java

```diff
@@ -0,0 +1,46 @@
+package duke.task;
+
+public abstract class Task {
+    protected String description;
+    protected String startDate;
+    protected String endDate;
+    protected boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        this.startDate = "";
+        this.endDate = "";
+        this.isDone = false;
+    }
+
+    public String getStatusIcon() {
+        return ((this.isDone) ? "X" : " ");
+    }
```

Comment:
Try to replace these string literals with named constants to better reflect their context